{
    "name": "villfa/TryAgain",
    "description": "The TryAgain library permits to call a function again if the result is wrong.",
    "version": "1.0.0",
    "type": "library",
    "keywords": [
        "library",
        "error handling"
    ],
    "homepage": "https://github.com/villfa/TryAgain",
    "license": "MIT",
    "authors": [
        {
            "name": "Fabien Villepinte",
            "email": "fabien.villepinte@gmail.com"
        }
    ],
    "require": {
        "php": "^7.0"
    },
    "require-dev": {
        "mockery/mockery": ">=0.8.0",
        "phpunit/phpunit": "^6.0"
    },
    "autoload": {
        "psr-0": { "TryAgain\\": "." }
    },
    "minimum-stability": "stable"
}
